FBAllocationTracker/NSObject+FBAllocationTracker.h

```diff
@@ -31,3 +31,12 @@
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
```

Comment:
This code could also be wrapped inside _INTERNAL_FBAT_ENABLED check, mind putting it below your code?